### ISSUE TITLE
Document set data method for updated data

### ISSUE THREAD

**Issue by andreabrambilla** [equinor/esv-intersection#236]

**Comment by andreabrambilla** [comment]
@ooystein can you provide more details about where you looked for this method?

**Comment by ooystein** [comment]
Sorry what are you asking for, not sure I have the full context here?
PS. Sven did the first implementations of our integration

**Comment by Brynjulf** [comment]
@ooystein, I can elaborate. I was interested to know where you were looking for documentation about `setData`, or any other methods for that matter. If it's hard to find the documentation, then we need to fix unless we just haven't gotten around to documenting it properly

**Comment by andreabrambilla** [comment]
this was initially reported by @stalenestas , he mentioned you two were working on the intersection and found out that one of the `setData` methods was not listed in the documentation

But yes, as Brynjulf says, any feedback on the documentation is more than welcome!

**Comment by ooystein** [comment]
Ok.

I continued on the work of Sven and started with a meeting with Ståle, so he told me about the setData before I even started looking for it. That's great documentation for ya! :smile: 

But having a quick look for the documentation of setData (and other APIs we use) in the code and docs it seem to be lacking.

I've used the storybook to get a quick understanding of the APIs, and feel that have worked fairly well. But some of the stories are a bit inconsistent with recent updates, so it could need some refreshing.
